config/bootscripts/boot-cubox.cmd

```diff
@@ -25,7 +25,7 @@ fi
 if test "${console}" = "display" || test "${console}" = "both"; then setenv consoleargs "console=tty1"; fi
 if test "${console}" = "serial" || test "${console}" = "both"; then setenv consoleargs "${consoleargs} console=${cons}"; fi
 if test "${earlycon}" = "on"; then setenv consoleargs "earlycon ${consoleargs}"; fi
-if test "${bootlogo}" = "true"; then setenv consoleargs "bootsplash.bootfile=bootsplash.armbian ${consoleargs}"; fi
```

Comment:
What if we leave legacy support under:
if test "${kernelbootlogo}" = "true"; then setenv consoleargs "bootsplash.bootfile=bootsplash.armbian ${consoleargs}"; fi

Reply:
But how to control it? Or just leave it as a user option?

Reply:
> Or just leave it as a user option?

Perhaps defaulting to plymouth but leave in the build system if user chooses KERNEL_BOOTLOGO that this is all they need to do. This means we need to leave support in boot scripts in some way.